modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -0,0 +1,79 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-create-kataconfig-rsource-cli_{context}"]
+= Creating the KataConfig custom resource using the CLI
+
+You must create one `KataConfig` custom resource (CR) to install the `kata` runtime on your nodes. Creating the `KataConfig` CR triggers the {sandboxed-containers-operator} to do the following:
+
+* Install the needed {op-system} extensions, such as QEMU and `kata-containers`, on your {op-system} node.
+* Ensure that the link:https://github.com/cri-o/cri-o[CRI-O] runtime is configured with the correct `kata` runtime handlers.
+* Create a `RuntimeClass` custom resource with the necessary configurations for additional overhead.
+
+[NOTE]
+====
+The Kata runtime is installed on all worker nodes by default. To install the `kata` runtime only on selected nodes, you must label the nodes first, and then create the `KataConfig` CR and include the label in the YAML file.
```

Comment:
This is generic, which parts of the YAML file? 

I see we iterate on it below, but this caught my attention with the info tag, so we could mention `kataConfigPoolSelector` part of the YAML file.

Reply:
Since this is just part of the introduction and I'm giving a general overview, I'm not sure we need to add that level of detail.